test/external/external_test_onnx_backend.py

```diff
@@ -107,6 +107,8 @@ def supports_device(cls, device: str) -> bool:
 backend_test.exclude('test_range_int32_type_negative_delta_expanded_cpu')
 
 # unsupported (strange) ops
+# TODO implement a Binarizer op? https://github.com/onnx/onnx/blob/67c456ba4747412afb44158a1a889c0fc3349641/onnx/reference/ops/aionnxml/op_binarizer.py#L13
+backend_test.exclude('test_ai_onnx_ml_binarizer_*')
```

Comment:
was this one passing before or is it a new test in different onnx version?

Reply:
the problem isn't accuracy it's dtypes

Reply:
can it be fixed here? https://github.com/tinygrad/tinygrad/blob/ca59054463b7d7567cf28d5ee81a008ed2ff8bab/extra/onnx_ops.py#L125